xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -463,7 +465,8 @@
       "properties": {
         "name": { "type": "string", "required": true },
         "role": { "type": "string", "required": true },
-        "artistid": { "type": "integer", "required": true }
+        "roleid": { "$ref": "Library.Id", "required": true },
```

Comment:
You're right. I thought this was the return type of `AudioLibrary.GetRoles` but it's not so it's fine.